src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-console.log('hello mate academy!');
+const bigImg = document.querySelector('.gallery__large-img');
+
+function replaceImg(e) {
+  const { className: elemClass } = e.target;
```

Comment:
зачем дописывать elemClass?
```suggestion
  const { target } = e;
```

Reply:
Я просто думал , что className это зарезервированное имя, так что просто вытягивал это свойство и записывал его в elemClass - переменную которая не должна конфликтовать